Dnd.Ddd.Model/Character/CharacterStates/Contract/CharacterState.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Dnd.Ddd.Common.Guard;
+
+
+namespace Dnd.Ddd.Model.Character.CharacterStates.Contract
+{
+    internal abstract class CharacterState
+    {
+        private static readonly IDictionary<CharacterStates, CharacterState> StatesFactoryMethods = new Dictionary<CharacterStates, CharacterState>
+        {
+            [CharacterStates.Completed] = new Completed(),
+            [CharacterStates.Draft] = new Draft()
+        };
```

Comment:
Given that this dictionary no longer contains factory methods (as we return object instances instead of delegates), this field can be renamed (although I am not sure what the name should be).

Reply:
I have changed it to StatesIntances for now

Reply:
Fair enough - resolving.